app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -2,188 +2,114 @@
+ * @param onStoryClick callback for when the user taps on this story.
  */
 @Composable
 fun PocketStory(
     @PreviewParameter(PocketStoryProvider::class) story: PocketRecommendedStory,
     client: Client,
-    modifier: Modifier = Modifier
+    onStoryClick: (PocketRecommendedStory) -> Unit,
 ) {
-    Column(
-        modifier
-            .size(160.dp, 191.dp)
-            .clip(RoundedCornerShape(4.dp))
-            .clickable { /* no-op */ }
-    ) {
-        Card(
-            elevation = 6.dp,
-            shape = RoundedCornerShape(4.dp),
-            modifier = Modifier.size(160.dp, 87.dp)
-        ) {
-            ImageLoader(
-                client = client,
-                // The endpoint allows us to ask for the optimal resolution image.
-                url = story.imageUrl.replace(
-                    "{wh}",
-                    with(LocalDensity.current) {
-                        "${160.dp.toPx().roundToInt()}x${87.dp.toPx().roundToInt()}"
-                    }
-                ),
-                targetSize = 160.dp
-            ) {
-                WithImage { painter ->
-                    Image(
-                        painter,
-                        modifier = Modifier.size(160.dp, 87.dp),
-                        contentDescription = "${story.title} story image"
-                    )
-                }
-
-                Placeholder {
-                    Box(
-                        Modifier.background(
-                            when (isSystemInDarkTheme()) {
-                                true -> Color(0xFF42414D) // DarkGrey30
-                                false -> PhotonColors.LightGrey30
-                            }
-                        )
-                    )
-                }
-
-                Fallback {
-                    Box(
-                        Modifier.background(
-                            when (isSystemInDarkTheme()) {
-                                true -> Color(0xFF42414D) // DarkGrey30
-                                false -> PhotonColors.LightGrey30
-                            }
-                        )
-                    )
-                }
-            }
-        }
-
-        Spacer(modifier = Modifier.height(8.dp))
-
-        CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
-            Text(
-                modifier = Modifier.padding(bottom = 2.dp),
-                text = story.publisher,
-                style = MaterialTheme.typography.caption,
-                maxLines = 1,
-                overflow = TextOverflow.Ellipsis
-            )
+    val imageUrl = story.imageUrl.replace(
+        "{wh}",
+        with(LocalDensity.current) { "${116.dp.toPx().roundToInt()}x${84.dp.toPx().roundToInt()}" }
+    )
+    ListItemTabLarge(
+        client = client,
+        imageUrl = imageUrl,
+        onClick = { onStoryClick(story) },
+        title = { TabTitle(text = story.title, maxLines = 3) },
```

Comment:
Maybe `TabTitle` should go in a new line to follow the same style as `TabSubtitleWithInterdot`. The extra room and spacing might help people to recognize that this is @content more easily.